src/patternfly/components/Login/login.scss

```diff
@@ -179,13 +184,31 @@
   align-items: center;
   padding: var(--pf-c-login__main-header--PaddingTop) var(--pf-c-login__main-header--PaddingRight) var(--pf-c-login__main-header--PaddingBottom) var(--pf-c-login__main-header--PaddingLeft);
 
-  .pf-c-title {
+  @media (max-width: $pf-global--breakpoint--md) {
+    grid-template-columns: 100%;
+  }
+
+  .pf-c-dropdown {
+    grid-column: 2 / 3;
+    grid-row: 1;
+
     @media (max-width: $pf-global--breakpoint--md) {
-      grid-column: -1 / 1;
+      grid-column: auto;
+      grid-row: auto;
     }
   }
 }
 
+.pf-c-login__main-header-desc {
+  margin-bottom: var(--pf-c-login__main-header-desc--MarginBottom);
+  font-size: var(--pf-c-login__main-header-desc--FontSize);
+  grid-column: 1 / -1;
+
+  @media (max-width: $pf-global--breakpoint--md) {
```

Comment:
can you explain why you didn't add the media query at the top where you define the variables?

Reply:
that's an oversight, updated.